crates/bevy_render/src/texture/mod.rs

```diff
@@ -10,6 +10,7 @@ pub(crate) mod image_texture_conversion;
 pub use self::image::*;
 #[cfg(feature = "hdr")]
 pub use hdr_texture_loader::*;
+
 pub use image_texture_loader::*;
```

Comment:
should this one (and its mod declaration above) be behind a feature too?

Reply:
There were some error enums there that I felt should be compiled regardless.